src/ReactRotatingText.js

```diff
@@ -47,6 +47,9 @@ class ReactRotatingText extends React.Component {
     if (output.length < text.length) {
       this._loop(loopingFunc, typingInterval);
     } else {
+	  if (typeof this.props.onTypingEnd == "function") {
+	  	this.props.onTypingEnd(this);
+	  }
```

Comment:
Looks like some inconsistent formatting, can you run `eslint --fix` to correct this?

Reply:
In my use case, I want to refer to `state.index` so I can select the image which matches the text. Do you have a better way of achieving this?